ApollosProject/apollos-apps
# fix: Remove support for multiple layers on PulseImage

## ✏️ Description
The multi-layer support for the PulseImage required us to manually set the fade on each layer by putting a colored overlay on top of each background layer to avoid opacity-based overlap.

This was causing some strange behavior when looking at the PulseImage being used overtop colored background.

In order to make sure that this component is the best visually, we've updated it to use only a single background layer for the time being. Support for anything beyond that can be added at a later date.

## 🔬 To Test
1. Run the storybook application
2. Check out the new `Basic Implementation` story

## 📸 Screenshots
| Feature | Simulator | Figma | Links |
| --- | --- | --- | --- |
| Updated Screenshots | ![CleanShot 2022-11-10 at 12 24 39](https://user-images.githubusercontent.com/45076058/201164403-907ca0ec-5670-4da2-8155-c0e1d483c05f.png) | ![CleanShot 2022-11-10 at 12 25 14](https://user-images.githubusercontent.com/45076058/201164592-24c852f2-fdf2-4825-93a0-defe1fe567e6.png) | [Link to Figma](https://www.figma.com/file/YHJLj8pdFxWG9npF2YmB3r/UI-Kit-2.0?node-id=4459%3A68805) |

## ⚠️ To-do Before Merge
- [x] Update Figma to reflect some design changes made in code

## 📝 Checklist
